server/src/report/latex.rs

```diff
@@ -0,0 +1,88 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2024 Aliaksei Bialiauski
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+use std::fs;
+use std::path::Path;
+
+/// Read LaTeX template.
+/// @todo #41:60min Add function for appending new content into the template.
+///  We need to create new function that will append input into the template,
+///  thus it will build a detailed report.
+/// # Arguments
+///
+/// * `path`: Template path
+///
+/// returns: String
+///
+/// # Examples
+///
+/// ```
+/// use crate::server::report::latex::template;
+/// let content = template(None);
+/// print!("{content}")
+/// ```
+pub fn template(path: Option<&str>) -> String {
```

Comment:
@h1alexbel i think in this case path should be required, you can add it to `Args` and set default value if nothing was provided to `--report` argument; `txt` would be fine for default value

Reply:
@l3r8yJ this function will be called only if cli parser will find non-empty `--report` option

Reply:
@h1alexbel can you add the puzzle or implementation?

Reply:
@l3r8yJ it's already added as puzzle in `cli/src/args.rs`